app/src/main/res/layout/activity_maps.xml

```diff
@@ -20,32 +20,35 @@
             android:orientation="horizontal"
             android:paddingTop="@dimen/maps_activity_radius_container_padding"
             android:paddingLeft="@dimen/maps_activity_radius_container_padding"
-            android:paddingRight="@dimen/maps_activity_radius_container_padding">
+            android:paddingRight="@dimen/maps_activity_radius_container_padding"
+            android:paddingBottom="@dimen/maps_activity_radius_container_padding"
+            >
 
             <TextView
-                android:layout_width="wrap_content"
+                android:layout_width="50dp"
```

Comment:
wrap_content is better because text can change in the future and then we will need to change it again. Return to wrap_content and change string resource to make it shorter.

Reply:
What do you want to be shorter
?

Reply:
Wrap content dont let the seek bar to take the empty space that left